evalml/data_checks/outliers_data_check.py

```diff
@@ -61,5 +66,8 @@ def get_IQR(df, k=2.0):
         has_outliers = ((X < iqr['lower_bound']) | (X > iqr['upper_bound'])).any()
         warning_msg = "Column '{}' is likely to have outlier data"
```

Comment:
@dsherry Good point! I was curious why this data check and not, then, updating all of our data checks to follow this pattern... but for certain data checks, we have different thresholds or percentages that we also report (ex: % null) while there isn't information that's lost here. This may make our outputs less consistent, but I agree that it could be easier to see one data check for all the outliers, so I'll update this one :)